### ISSUE TITLE
CDK CLI: Allow deploy CLI details to persist

### ISSUE THREAD

**Issue by evankroske** [aws/aws-cdk#23712]
### Describe the bug

When CDK encounters an error while deploying one of my stacks to cfn, it shows an error messages with details of the error, and then later deletes that error message from the console and shows me a general message like "The stack named X failed to deploy: UPDATE_ROLLBACK_COMPLETE". This makes it very hard to debug slow CDK deploys.

### Expected Behavior

I expect the new error message to be printed after the source error message.

### Current Behavior

The new error message replaces the source error message.

### Reproduction Steps

I think any CDK code that triggers a cfn rollback will see this behavior. For example, you could delete an export used by another cfn stack:

https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-stack-export-name-error/

### CDK CLI Version

2.59.0 (build b24095d)

### Node.js Version

14

### OS

Amazon Linux 2

### Language

Typescript

**Comment by peterwoodworth** [comment]
Yeah this can make things harder to debug than they have to be. I'm pretty sure this is a consequence of CloudFormation's api, but I'm sure we could configure this such that all the deploy details persist, thanks for reporting!

**Comment by mrgrain** [comment]
Yes, duplicate.
